### DISCUSSION TITLE
Herramientas para pruebas

### DISCUSSION THREAD

**Discussion by GustavoGutierrez21** [Launch-X-Latam/MisionFrontEnd#77] [Q&A]
Compañeros ya experimentados, ¿qué herramientas para ingeniería de pruebas recomiendan o conocen que sean buenos?

**Comment by manuel260300** [comment]
Si te refieres a pruebas unitarias de codigo esta bueno JEST

**Comment by JosafatJimenezB** [comment]
si te entendi bien, para realizar pruebas unitarias existen diversas herramientas como lo es **Jest**

**Comment by Emilio17MC** [comment]
Depende mucho de lo que estés haciendo, pero al menos para el caso de JavaScript una librería de aserciones puede ser [JEST](https://github.com/facebook/jest) o Chai. Si estás usando React podrías usar  _react-testing-library_, antiguamente se hacía mediante _Enzyme_ y JEST, pero me parece que desde la versión 17 de React dejaron de darle soporte.

Por mencionar otras herramientas está Cypress, Postman (de alguna forma se puede hacer testing), Selenium, JIRA puede ser útil para gestionar nuestras pruebas en un proyecto, [Karma](https://github.com/karma-runner/karma), Azure Test Plans, TestRail.

**Comment by doguedogue** [comment]
JUnit para Java ☕
